easybuild/easyconfigs/t/Trinity/Trinity-2.0.6-goolf-1.4.10.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
```

Comment:
@boegel compilation instructions in their website say just "make && make plugins"

maybe this has changed since previous versions and that's why before a custom easyblock was needed?